packages/astro-feather-icons/README.md

```diff
@@ -1,45 +1,91 @@
-# astro-feather-icons
+# Astro Feather Icons
 
-[GitHub](https://github.com/edazpotato/astro-feather-icons) |
-[NPM](https://npmjs.com/package/astro-feather-icons)
+Feather is a collection of simply beautiful open source icons.
+Each icon is designed on a 24x24 grid with an emphasis on simplicity, consistency, and flexibility.
 
-## Install
+```shell
+npm install astro-feather-icons
+```
 
-```bash
-npm i astro-feather-icons --save
+```shell
 pnpm i astro-feather-icons
 ```
 
+```shell
+yarn add astro-feather-icons
+```
+
+[GitHub](https://github.com/edazpotato/astro-feather-icons) |
+[NPM](https://npmjs.com/package/astro-feather-icons)
+
+
+
 ## Usage
 
+All of the icons are available from a single import, and only the icons that are used will be added to the page.
+
 ```astro
 ---
-import { AirplayIcon, AtSignIcon, ... } from 'astro-feather-icons'
```

Comment:
Is this style of import still possible with your changes?

Reply:
Yes, this change still allows individual named imports.

Note: The names would not include the 'Icon' suffix by default:

```js
import { AirPlay } from 'astro-feather-icons';
```

Therefore, to rename them:

```js
import { AirPlay as AirPlayIcon } from 'astro-feather-icons';
```

Another note: To have it _both_ ways would require duping the exports.

Reply:
Done in 32d4d5920ec4625a79b1702021215c301f03d060